pkgs/applications/misc/tdrop/default.nix

```
@@ -0,0 +1,35 @@
+{ stdenv, lib, fetchFromGitHub, makeWrapper
+, xwininfo, xdotool, xprop }:
+
+let
+  version = "2019-01-02";
+in
+stdenv.mkDerivation {
```

```suggestion
stdenv.mkDerivation rec {
```

This allows you to self reference so you can move `version` into the package description.